### DISCUSSION TITLE
Can I use a browser to access the UI of a packaged tauri app?

### DISCUSSION THREAD

**Discussion by xiaguangbo** [tauri-apps/tauri#9495] [Q&A] [answered]
such as access tauri across devices within a local area network

**Comment by FabianLars** [comment, marked as answer]
By default the UI is only accessible inside the app itself. You can expose it on a localhost http server with https://github.com/tauri-apps/plugins-workspace/tree/v1/plugins/localhost though. If you also need tauri's apis to work you'll need https://github.com/tauri-apps/tauri-invoke-http/ too.

**Reply by xiaguangbo** [reply]
thanks
